api/src/main/resources/messages.properties

```diff
@@ -80,34 +83,63 @@ edtriageapp.formHistory=Form History
 
 # symptoms that need translations
 edtriageapp.117617AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=Pregnancy & trauma or vaginal bleeding
-edtriageapp.120977AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=Burn - facial/inhalation
```

Comment:
Just to confirm @lnball  we want to maintain all these English translations in Haiti, or there any we want toc change everywhere?  Are this all the "official" South African triage English names and so we want to keep a copy of them for that reason?